tests/snapshots/tests.contrib.yaaredis.test_yaaredis.test_unicode.json

```diff
@@ -0,0 +1,28 @@
+[[
+  {
+    "name": "redis.command",
+    "service": "redis",
+    "resource": "GET \ud83d\ude10",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "redis",
+    "meta": {
+      "out.host": "localhost",
+      "redis.raw_command": "GET \ud83d\ude10",
```

Comment:
this is what we expect because JSON doesn't support unicode... ?

Reply:
definitely should.. https://tools.ietf.org/id/draft-ietf-json-rfc4627bis-09.html

Is this the resulting output from the trace client or is this possibly happening on the test agent?

Reply:
Since this appears to just be how the test agent encodes it and it is getting encoded fine by the encoder/writer, then this is all good!